### DISCUSSION TITLE
can not turn the led on on rasberry pi pico W

### DISCUSSION THREAD

**Discussion by manes6969** [tabemann/zeptoforth#39] [Q&A] [answered]
Hi ,
I just discovered zeptoforth.
I was testing and tried to switch the led on on the pico w. the led is connected to gpio0.
So I did this 
pin import
0 output-pin
high 0 pin!
no led on ???
If I then type 0 pin@ i receive a zero , so it stayed low.
What am I missing here ?
Anyone has any idea ??
Regards,
Ronny

**Comment by manes6969** [comment]
Tried on the rpi pico and it worked with on green led!
Why not on the pico W i don't know yet.
Looking further.
Ronny

**Comment by manes6969** [comment, marked as answer]
Found the problem ! The led is connected to the wireless chip on gpio0 , not gpio0 of the rp2040.
So problem solved :-)

**Comment by tabemann** [comment]
I should note that the Pico W is not really supported at the present, because using its CYW43 radio chip, which as you say is necessary to exercise its LED, is not supported due to a lack of a  CYW43 driver (as I would have to derive it from Damien George's CYW43 driver, and there are intellectual property issues with that driver due to Damien's not releasing a FLOSS driver).

**Reply by manes6969** [reply]
ok thanks. I will use the normal pico instead ;-)
